acceptance/tests/facts/networking_facts.rb

```diff
@@ -0,0 +1,80 @@
+test_name 'FACT-1361 - C59029 networking facts should be fully populated'
+
+#
+# This test is intended to ensure that networking facts resolves
+# as expected in AIO across supported platforms.
+#
+
+skip_test "Networking fact test is confined to AIO" if @options[:type] != 'aio'
+
+@ip_regex = /^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/
+
+expected_networking = {
+    "networking.dhcp"     => @ip_regex,
+    "networking.ip"       => @ip_regex,
+    "networking.ip6"      => /[a-f0-9]+:+/,
+    "networking.mac"      => /[a-f0-9]{2}:/,
+    "networking.mtu"      => /\d+/,
+    "networking.netmask"  => /\d+\.\d+\.\d+\.\d+/,
+    "networking.netmask6" => /[a-f0-9]+:/,
```

Comment:
i think these are always `2001:0db8:0123:4567:89ab:cdef:1234:5678`  
as in 8 sets of 4 hex encoded chars separated by colons